Source/Shapes/Shape.cpp

```diff
@@ -0,0 +1,5 @@
+#include "Ashkal/Shapes/Shape.hpp"
+
+using namespace Ashkal;
+
+Shape::~Shape() {}
```

Comment:
Eliminate this file.

Use an explicitly defaulted destructor:

https://en.cppreference.com/w/cpp/language/member_functions#Special_member_functions